test_conformance/math_brute_force/unary_two_results.cpp

```diff
@@ -253,8 +252,7 @@ int TestFunc_Float2_Float(const Func *f, MTdata d)
             if ( gTestFastRelaxed && strcmp(f->name,"sincos") == 0 )
             {
               float pj = *(float *)&p[j];
-              if(fabs(pj) > M_PI)
-                p[j] = NAN;
+              if (fabs(pj) > M_PI) p[j] = 0;
```

Comment:
I think I'd rather we don't try to fix these in this PR. See https://github.com/KhronosGroup/OpenCL-CTS/pull/699.